package.json

```diff
@@ -27,9 +27,11 @@
     "activationEvents": [
         "onCommand:extension.vsKubernetesCreate",
         "onCommand:extension.vsKubernetesCreateFile",
+        "onCommand:extension.vsKubernetesAddWatch",
```

Comment:
I realised we need to add the "resources to watch" config to `contributes.configuration`